src/dfind/dfind.c

```diff
@@ -582,12 +699,11 @@ int main (int argc, char** argv)
     }
 
 
+    pred_commit(pred_head);
```

Comment:
We have pred_commit in here twice.  This one was added, but there is another call higher up.  Is that intentional?

Reply:
@adammoody fixed in #339, no I removed one, good catch